src/Images.jsx

```diff
@@ -151,7 +151,7 @@ class Images extends React.Component {
         let toggleIntermediate = "";
         if (interm) {
             toggleIntermediate = <span className="listing-action">
-                <Button variant="link" onClick={() => this.setState({ intermediateOpened: !intermediateOpened, isExpanded: true })}>
```

Comment:
This is needed for the case that there are not images, only intermediate. In this case. clicking show intermediate images, will make the expandable section to appear. I think it makes sense to make it appear expanded.

Reply:
ah right, makes sense!